setup.py

```diff
@@ -42,6 +42,7 @@
               'snapcraft.internal',
               'snapcraft.internal.cache',
               'snapcraft.internal.deltas',
+              'snapcraft.internal.new_sources',
```

Comment:
still call this `sources`, I don't see a reason for `new_sources`. The migration could be transparent and if it isn't it doesn't matter *as much* as it lives in `internal`